apps/openmw/mwgui/class.cpp

```diff
@@ -12,8 +12,13 @@
 
 #include "../mwworld/esmstore.hpp"
 
+#include "../../../components/misc/rng.hpp"
```

Comment:
Why not include it like ``<components/misc/rng.hpp>`` like anything else does?

Reply:
Oh, I didn't know the include paths were like that. I'll do that.